server/meshmodel/components/core/WASMFilter.json

```diff
@@ -0,0 +1,33 @@
+    },
+    "model": {
+        "name": "core",
+        "version": "v1.0.0",
+        "displayName": "Core",
+        "category": {
+            "name": "Orchestration & Management",
+            "metadata": null
+        },
+        "metadata": {
+            "svgColor": "<svg xmlns=\"http:\/\/www.w3.org\/2000\/svg\" id=\"Layer_1\" data-name=\"Layer 1\" viewBox=\"0 0 134.95 135.02\" height=\"20\" width=\"20\"><defs><style>.cls-1{fill:#00d3a9}.cls-2{fill:#00b39f}<\/style><\/defs><path class=\"cls-1\" d=\"M69.49 31.82v32.25l27.95-16.18-27.95-16.07zm0 38.99v32.41L97.7 87.09 69.49 70.81z\"\/><path class=\"cls-2\" d=\"M65.47 63.85V32.09l-27.6 15.83 27.6 15.93zM10.1 103.1a67.79 67.79 0 0 0 21.41 21.55V90.71Zm55.37-.04V71.05L37.8 87.07l27.67 15.99z\"\/><path class=\"cls-1\" d=\"m35.54 122.63 28.02-16.02-28.02-16.2v32.22z\"\/><path class=\"cls-2\" d=\"M99.61 122.8V90.63l-27.98 16 27.98 16.17zM127 99.37a67.22 67.22 0 0 0 7.91-28.94l-29.13 16.68Z\"\/><path class=\"cls-1\" d=\"m103.64 83.69 28.12-16.08-28.12-16.16v32.24z\"\/><path class=\"cls-2\" d=\"M99.61 44.5V12.52L71.76 28.49 99.61 44.5zm0 39.05V51.28L71.7 67.44l27.91 16.11zm-32.13 51.47h.01-.01z\"\/><path class=\"cls-1\" d=\"M35.54 51.22v32.51l28.12-16.28-28.12-16.23z\"\/><path d=\"M65.47 0a67.2 67.2 0 0 0-29.64 7.83l29.64 17Z\" class=\"cls-2\"\/><path class=\"cls-1\" d=\"M35.54 12.3v32.32l28.14-16.14L35.54 12.3z\"\/><path d=\"M31.51 10.34A67.89 67.89 0 0 0 10.1 31.89l21.41 12.36Z\" class=\"cls-2\"\/><path d=\"M99.43 8a67.23 67.23 0 0 0-29.94-8v25.15ZM0 69.87a67.27 67.27 0 0 0 8.07 29.76l21.69-12.56Zm8.07-34.5A67.16 67.16 0 0 0 0 65l29.79-17.09Z\" class=\"cls-1\"\/><path d=\"M35.78 127.13A67.13 67.13 0 0 0 65.47 135v-24.85Z\" class=\"cls-2\"\/><path d=\"M124.92 32a67.9 67.9 0 0 0-21.28-21.52V44.3Zm-21.28 92.54A68 68 0 0 0 125 102.86l-21.36-12.34Z\" class=\"cls-1\"\/><path d=\"M135 64.81a67.06 67.06 0 0 0-8-29.35l-21.51 12.42Z\" class=\"cls-2\"\/><path d=\"M69.49 135a67.12 67.12 0 0 0 29.63-7.83L69.49 110Z\" class=\"cls-1\"\/><path class=\"cls-2\" d=\"M31.51 83.44V51.56L3.83 67.43l27.68 16.01z\"\/><\/svg>",
```

Comment:
@Revolyssup do you know why were defining model information twice? We see this duplication in a number of other components.